rules/X++/ComplexityIndentationCombined.xq

```diff
@@ -0,0 +1,87 @@
+(: Combines Complexities and MethodIdentationLevel queries :)
+(: to Find methods on class or table types with a complexity level above 30 :)
+(: Error if maximum indentation of the method is above 2, warning otherwise :)
+(: @Language Xpp :)
+(: @Author pvillads@microsoft.com :)
```

Comment:
I don't think we need to put this? Git history will show this kind of stuffs

Reply:
This information (in (: .. :)) is nice to have since the Socratex Explorer will show this in a popup when looking at the rules pane.